WisdomAndGrace/client/src/index.js

```diff
@@ -5,19 +5,28 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import * as firebase from "firebase/app";
+import { Spinner } from "reactstrap";
 
 const firebaseConfig = {
   apiKey: process.env.REACT_APP_API_KEY,
 };
 firebase.initializeApp(firebaseConfig);
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Spinner className="app-spinner dark"/>,
```

Comment:
I imagine you ran this, so let me know, but in my experience it's taken firebase almost no time to read the auth state. Is this spinner showing up for more than a fraction of a second? In which case, do you want to still keep it?